### PR TITLE
refactor(store-engine): Convert Node.js tests to TypeScript

### PR DESCRIPTION
**Pull request by bennyn**
## Pull Request Checklist

- [x] My code is covered by tests
- [x] I will [merge the PR as breaking change](https://github.com/wireapp/wire-web-packages/wiki/Releases#create-major-release), if the API contract changes

### CODE REVIEW

File: packages/store-engine/src/main/engine/FileEngine.test.node.ts [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import {FileEngine} from '../engine';
```
Comment (ffflorian) [inline packages/store-engine/src/main/engine/FileEngine.test.node.ts]: Let's add an empty line here to distinguish the imports

```suggestion

import {FileEngine} from '../engine';
```

File: tsconfig.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "jsx": "react",
     "module": "commonjs",
     "moduleResolution": "node",
-    "noEmitOnError": true,
+    "noEmitOnError": false,
```
Comment (ffflorian) [inline tsconfig.json]: Why did you change this?

Comment (bennyn) [reply tsconfig.json]: To get error output from `ts-node`.
